src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -503,4 +503,31 @@
   <data name="EnableAnalyzerReleaseTrackingRuleDescription" xml:space="preserve">
     <value>Enabling release tracking for analyzer packages helps in tracking and documenting the analyzer diagnostics that ship and/or change with each analyzer release. See details at https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md.</value>
   </data>
+  <data name="DefineDiagnosticTitleCorrectlyDescription" xml:space="preserve">
+    <value>The diagnostic title is recommended to be a short sentence without any period ('.)' nor any line return ('\r', '\n' or '\r\n').</value>
```

Comment:
```suggestion
    <value>The diagnostic title is recommended to be a short sentence without any period ('.') or any line return ('\r', '\n' or '\r\n').</value>
```

Reply:
Probably we don't specify a description string for descriptors where its content would be identical to the message.